crates/driver/openapi.yml

```diff
@@ -35,6 +94,25 @@ components:
       description: Amount of an ERC20 token. 256 bit unsigned integer in decimal notation.
       type: string
       example: "1234567890"
+    Auction:
+      description: |
+        Auction information.
+
+        Contains `id`, `orders`, `prices` from the order book api's auction endpoint.
+
+        TODO: Document the object here. Do we need `block` and `latestSettlementBlock`?
```

Comment:
Since `autopilot` is cutting the auctions **and** starting the `solve` requests, I don't think `latestSettlementBlock` will be needed anymore.

`block` makes sense for documentation purposes and is very useful historic data to have ("this is the block that the protocol thinks we are on")